Formula/rtl_433.rb

```diff
@@ -0,0 +1,42 @@
+class Rtl433 < Formula
+  desc "Program to decode radio transmissions from devices"
+  homepage "https://github.com/merbanan/rtl_433"
+  url "https://github.com/merbanan/rtl_433/",
+    using:    :git,
```

Comment:
```suggestion
  url "https://github.com/merbanan/rtl_433.git",
```

Reply:
The program version is formed based on the last tag of the git repository.
If I use tarball, the output of the program version will return null.

This code is a formatted version of the program:
https://github.com/merbanan/rtl_433/blob/master/cmake/Modules/GetGitRevisionDescription.cmake